.werft/installer-tests.ts

```diff
@@ -369,10 +376,13 @@ export async function installerTests(config: TestConfig) {
 }
 
 function runIntegrationTests() {
-    werft.phase("run-integration-tests", "Run all existing integration tests");
+    const testComponent: string = randomize(["WORKSPACES", "IDE", "WEBAPP"])
+    werft.phase(`run-${testComponent}-integration-tests`, `Run all ${testComponent} integration tests`);
+    console.log(`Running ${testComponent} tests`)
     const slackAlerts = new Map<string, string>([]);
-    for (let test in TESTS) {
-        const testPhase = TESTS[test];
+    const testSuite = eval(`${testComponent}_TESTS`)
```

Comment:
Fixed. Apologies, it is my first time in the javascript world :sweat_smile:

Reply:
No worries! And likewise, apologies if I got jumpy; things like `eval` totally make sense in a lot of other contexts (**cough** bash) because there aren't better tools; while they look reasonable in Javascript/Typescript the risk associated with eval might not be visible until _after_ a CVE comes out. I appreciate you chasing this down! ✨